### PR TITLE
fix tree view item visual bug on expansion

### LINKED ISSUE #5166: fix: TreeView has initial visual glitch when expanding tree groups
# 🐛 Bug Report

When expanding a tree group, you can see a quick rendering glitch for all of the child items under that group when they first appear. Successive collapsing/expanding of the same group does not have the glitch.

## 💻 Repro or Code Sample

I used the following snippet on your [FAST Tree View example](https://explore.fast.design/components/fast-tree-view) site:

```
<fast-tree-view>
    <fast-tree-item>Tree item 1<fast-tree-item slot="item">Test</fast-tree-item>
    </fast-tree-item>
    <fast-tree-item>Tree item 1<fast-tree-item slot="item">Item 2</fast-tree-item>
    </fast-tree-item>
    <fast-tree-item>Tree item 1<fast-tree-item slot="item">Item 3</fast-tree-item>
    </fast-tree-item>
    <fast-tree-item>Tree item 1<fast-tree-item slot="item">aasdf</fast-tree-item>
        <fast-tree-item slot="item">wer</fast-tree-item>
        <fast-tree-item slot="item">sfdh</fast-tree-item>
        <fast-tree-item slot="item">we34</fast-tree-item>
        <fast-tree-item slot="item">rewtret</fast-tree-item>
    </fast-tree-item>
    <fast-tree-item>Tree item 1
        <fast-tree-item slot="item">
            serfgsreg
        </fast-tree-item>
    </fast-tree-item>
    <fast-tree-item>Tree item 2</fast-tree-item>
</fast-tree-view>
```

To get around some of the initial wonky behavior the tree seems to exhibit on this example site (like the tree won't update with all  of the set tree items after copy-pasting the above snippet), you may have to update one of the text fields of one of the tree items.

## 🤔 Expected Behavior

There should be no glitch when expanding a tree group for the first time.

## 😯 Current Behavior

TreeGlitch

## 🔦 Context

The glitch is noticeable, and doesn't make for the best impression for a user of the tree.

## 🌍 Your Environment

Windows Desktop/Chrome.

### PR DESCRIPTION
**Pull request by chrisdholt**
# Pull Request

## 📖 Description
When expanding groups of tree items, there is a very slight issue where the DOM is conditionally loaded but styles are not fully set. This PR adds a very, very quick animation of opacity to manage the initial load of both DOM and styles which appears to solve the issue.

### 🎫 Issues
closes #5166

## 👩‍💻 Reviewer Notes
If reviewing and curious about the ever so slight, likely imperceptible value being used for the transition - please validate using the screen capture in the issue and validate against the fix.

### General

- [x] I have included a change request file using `$ yarn change`
- [ ] I have added tests for my changes.
- [x] I have tested my changes.
- [ ] I have updated the project documentation to reflect my changes.
- [x] I have read the [CONTRIBUTING](https://github.com/Microsoft/fast/blob/master/CONTRIBUTING.md) documentation and followed the [standards](https://www.fast.design/docs/community/code-of-conduct/#our-standards) for this project.

### Component-specific

- [ ] I have added a new component
- [ ] I have modified an existing component
- [ ] I have updated the [definition file](https://github.com/Microsoft/fast/blob/master/packages/web-components/fast-components/CONTRIBUTING.md#definition)
- [ ] I have updated the [configuration file](https://github.com/Microsoft/fast/blob/master/packages/web-components/fast-components/CONTRIBUTING.md#configuration)

### CODE REVIEW

File: packages/web-components/fast-components/src/tree-item/tree-item.styles.ts [hunk 1]
```diff
@@ -156,7 +165,6 @@ export const treeItemStyles: FoundationElementTemplate<ElementStyles, TreeItemOp
     }
 
     .items {
-        display: none;
```
Comment (chrisdholt) [inline packages/web-components/fast-components/src/tree-item/tree-item.styles.ts:159]: These are being conditionally rendered depending on when the tree is expanded or whether we the conditional flag is set, so I've removed this line of code as it seems unnecessary.

File: packages/web-components/fast-components/src/tree-item/tree-item.styles.ts [hunk 2]
```diff
@@ -210,7 +218,9 @@ export const treeItemStyles: FoundationElementTemplate<ElementStyles, TreeItemOp
     }
 
     :host([expanded]) > .items {
-        display: block;
```
Comment (chrisdholt) [inline packages/web-components/fast-components/src/tree-item/tree-item.styles.ts:213]: Per the above comment, this is already a block element - with the removal of above, this becomes redundant.

File: packages/web-components/fast-components/src/tree-item/tree-item.styles.ts [hunk 3]
```diff
@@ -94,6 +94,15 @@ export const treeItemStyles: FoundationElementTemplate<ElementStyles, TreeItemOp
     definition
 ) =>
     css`
+    @keyframes treeItemLoading {
```
Comment (EisenbergEffect) [inline packages/web-components/fast-components/src/tree-item/tree-item.styles.ts:97]: This might be worth adding a code comment to for future readers.

**Review by EisenbergEffect — APPROVED** [review]
Interesting hack.

File: packages/web-components/fast-components/src/tree-item/tree-item.styles.ts (hunk 3 above)
Comment (chrisdholt) [reply packages/web-components/fast-components/src/tree-item/tree-item.styles.ts:97]: I thought about that - I'll add a note here.

**Review by scomea — APPROVED** [review]

**Pull request merged**